R/colors.R

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8-unix -*-
+library(tidyverse)
+
+# Colors from:
+# https://www.hsl.fi/sites/default/files/tyyliopas/hsl-varit-2015.pdf
+# HSL: September 2018
+
+hsl_colors <- c(
+  # brand
+  blue = "#007AC9",
+  lightblue = "#bee4f8",
+  pink = "#f092cd",
+  lightpink = "#f4deec",
+  green = "#64be1e",
+  lightgreen = "#d0e6be",
+  gray = "#999999",
+  lightgray = "#dddddd",
+  red = "#dc0451",
+  white = "#FFFFFF",
+  # transit modes
+  tram = "#00985f",
+  ferry = "#00b9e4",
+  metro = "#ff6319",
+  train = "#8c4799",
+  lrt = "#00b2a9"
+)
```

Comment:
We will use mainly [MAL brand colors](https://tyyliopas.hsl.fi/d/h8JR9dHeqfgd/braendi#/mal/mal2023-maankaeyttoe-asuminen-ja-liikenne-ilme) but these are great too because MAL colors are quite limited. I can modify these later if I see fit. It is great that these helper functions exist already!